target/scripts/helper-functions.sh

```diff
@@ -209,6 +209,29 @@ function _monitored_files_checksums
 }
 export -f _monitored_files_checksums
 
+# ? --------------------------------------------- General
+
+function _obtain_hostname_and_domainname
+{
+  if [[ -n "${OVERRIDE_HOSTNAME}" ]]
+  then
+    export HOSTNAME="${OVERRIDE_HOSTNAME}"
+    export DOMAINNAME="${DOMAINNAME:-${HOSTNAME#*.}}"
+    if [[ ! "${DOMAINNAME}" =~ .*\..* ]] # Handle situations where the hostname is name.tld and hostname -d ends up just showing "tld"
```

Comment:
I'd like to see the comments above and cut after 80 characters to follow our style guide:)